.github/workflows/publish.yml

```diff
@@ -43,25 +43,28 @@ jobs:
         uses: docker/metadata-action@v5
         with:
           images: ${{ env.REGISTRY }}/${{ github.repository }}
-      - name: Build Docker Image
+      - name: Build and Push Docker Image
         uses: docker/build-push-action@v5
         with:
           context: .
-          load: true
-          tags: ${{ env.REGISTRY }}/nasa-ammos/aerie-ui:${{ github.sha }}
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+
+  scan:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: read
+    needs: publish
+    steps:
+      - name: sleep to let new :develop image settle
+        run: sleep 1m
       - name: Scan aerie-ui Docker image
         uses: aquasecurity/trivy-action@master
         with:
-          image-ref: ${{ env.REGISTRY }}/nasa-ammos/aerie-ui:${{ github.sha }}
+          image-ref: ${{ env.REGISTRY }}/nasa-ammos/aerie-ui:develop
```

Comment:
When reviewing with @skovati we realized that, if the newly-pushed `develop` tag is *not* ready to be pulled at this point (ie. if the 1m sleep wasn't long enough) - this could silently scan the *previous* version & we would never know. Plan is to restructure to be more similar to the original approach, but not have the scan step return an error code, which should allow the publish to still continue.

In future, we'd like to improve tagging of published images so we have a more consistent identifier.

Reply:
Change has been made @dandelany